sys-block/rts_pstor/rts_pstor-1.10_p20160103.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-mod
+
+GIT_COMMIT="c8313abffe083ac63bf76d2cc90d3edf5b2d1188"
+
+DESCRIPTION="PCIE RTS5209 card reader driver for Linux"
+HOMEPAGE="https://github.com/gexplorer/RTS5209-linux-driver"
+SRC_URI="https://github.com/gexplorer/RTS5209-linux-driver/archive/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/RTS5209-linux-driver-${GIT_COMMIT}"
+
+MODULE_NAMES="rts_pstor(misc/drivers/scsi)"
+BUILD_TARGETS="default"
+CONFIG_CHECK="~!MISC_RTSX_PCI"
+ERROR_MISC_RTSX_PCI="CONFIG_MISC_RTSX_PCI: The in-kernel driver rtsx_pci is configured, which may have the same functionality than this driver. Make sure that your kernel loads only one of both drivers - use a module blacklisting."
```

Comment:
Maybe install a modprobe.d file for it?

Reply:
Yes, can do that, but isn't that a little bit too patronising?

Reply:
I think that's what nvidia-drivers are doing. Principle of least surprise, I guess.

Reply:
automatic blacklist for in-tree module is now in ebuild